### ISSUE TITLE
Logout and Delete confirmation message and welcome greeting message

### ISSUE THREAD

**Issue by shrey1608** [mann27/Albums#53]
I should add the confirmation for delete button too as if by mistake user clicks it , it would delete it so it would be useful. what say @mann27 @ExpressHermes ?

**Comment by ExpressHermes** [comment]
Good idea. When the button is clicked, make a modal which will be triggered and ask for confirmation.

**Comment by shrey1608** [comment]
okay!

**Comment by shrey1608** [comment]
![image](https://user-images.githubusercontent.com/45357209/98024265-0f5cc900-1e2e-11eb-8811-ba9947ef0418.png)
Is is looking fine @mann27 @ExpressHermes

**Comment by shrey1608** [comment]
please reply @mann27 @ExpressHermes

**Comment by mann27** [comment]
Yeah looks fine!

**Comment by ExpressHermes** [comment]
Looks good!

**Comment by shrey1608** [comment]
hey @ExpressHermes when i try to delete the album the album which is added first means i have added 3 albums named 1 2 3 in this order and when i try to delete 3rd  1st gets deleted any reason why this is happening? 

Edit
I think it doesnt know which album's delete is clicked and isnt getting the album id so its deleting the albums in sequence.
```
<a  class="btn btn-danger" role="button" data-toggle="modal" data-target="#deleteModal">Delete</a>
```
```
<!--Modal-->
        <div class="modal fade" id="deleteModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
          <div class="modal-dialog" role="document">
            <div class="modal-content">
              <div class="modal-header">
                <h5 class="modal-title" id="exampleModalLabel">Delete</h5>
                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                  <span aria-hidden="true">&times;</span>
                </button>
              </div>
              <div class="modal-body">
                Are you sure you want to Delete?
              </div>
              <div class="modal-footer">
                <button type="button" class="btn btn-danger" data-dismiss="modal">Cancel</button>
                <a class="btn btn-primary" href="{%url 'fpage:album-delete' album.id  %}">Confirm</a>
              </div>
            </div>
          </div>
        </div>
        <!--Modal-->
```

**Comment by shrey1608** [comment]
hey @ExpressHermes have you worked with sweetalert2?

**Comment by ExpressHermes** [comment]
Bootstrap has a way to send data to modals using custom JavaScript. Read the docs about it. Use `this` keyword when modal is triggered and give a `data-album-id` which can be accessed using the this keyword. Just like I did with ajax like feature.

**Comment by shrey1608** [comment]
```
function remove(){
                Swal.fire({
                    title: 'Are you sure?',
                    text: "You won't be able to revert this!",
                    icon: 'warning',
                    showCancelButton: true,
                    confirmButtonColor: '#3085d6',
                    cancelButtonColor: '#d33',
                    confirmButtonText: 'Yes, delete it!'
                    }).then((result) => {
                    if (result.isConfirmed) {
                        
                        url: $(elem).attr('data-bookmark-url'),    // url where request need to be sent
                        data: {
                        'album_id': $(elem).attr('data-album-id'),    // album id
                                },
                        }
                    })
            }
```
I tried this @ExpressHermes  but its not working

**Comment by ExpressHermes** [comment]
Make a commit, I will check it out and get back to you

**Comment by ExpressHermes** [comment]
Make a commit, I will check it out and get back to you

**Comment by shrey1608** [comment]
i have made it please have a look @ExpressHermes

**Comment by shrey1608** [comment]
Solved in #55
